pavelib/utils/test/utils.py

```diff
@@ -17,7 +17,7 @@ def clean_test_files():
     """
     Clean fixture files used by tests and .pyc files
     """
-    sh("git clean -fqdx test_root/logs test_root/data test_root/staticfiles test_root/uploads")
+    sh("git clean -fqdx test_root/logs test_root/data test_root/uploads")
```

Comment:
Have you tried updating [this line](https://github.com/edx/edx-platform/blob/master/pavelib/utils/test/suites/bokchoy_suite.py#L60) to also check that it is not using fasttest?